<ui version="4.0" >
 <class>ImportGrammarWorkingPage</class>
 <widget class="QWidget" name="ImportGrammarWorkingPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lbDescription" >
     <property name="text" >
      <string>Die Texte werden analysiert.

Dies kann je nach Umfang der Texte und der Wortliste einige Zeit dauern.

Bitte beachten Sie das die Wörter mit Hilfe der Wortliste aufgelöst werden. Wörter die nicht in der Wortliste vorkommen, können nicht aufgelöst werden und die Sätze in denen sie vorkommen können nicht interpretiert werden.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Gesamt:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbTotal" >
     <property name="value" >
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Datei:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="pbProgress" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbCurrentAction" >
     <property name="font" >
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text" >
      <string>Warten...</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
